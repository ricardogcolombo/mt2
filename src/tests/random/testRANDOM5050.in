50 50
1 24 86 46 128
1 20 45 28 11
1 22 24 27 17
1 46 85 22 27
1 44 61 30 112
1 21 84 46 20
1 27 12 6 130
1 37 95 24 43
1 2 104 42 87
1 11 38 47 74
1 28 27 14 47
1 1 62 38 35
1 45 55 38 8
1 28 67 1 14
1 45 66 12 36
1 4 60 6 38
1 30 69 37 102
1 9 58 16 20
1 47 62 21 96
1 43 68 32 71
1 46 110 12 127
1 10 38 11 55
1 24 129 6 98
1 18 59 6 77
1 13 96 37 114
1 3 23 31 98
1 13 121 38 129
1 7 75 20 48
1 37 22 23 0
1 49 95 47 114
1 49 47 22 15
1 8 124 26 43
1 21 70 48 33
1 4 84 29 99
1 27 113 13 98
1 14 56 25 58
1 42 102 7 56
1 18 103 46 83
1 30 98 17 57
1 5 47 21 0
1 18 46 8 15
1 1 91 32 71
1 34 129 20 69
1 32 74 22 89
1 33 0 8 100
1 42 35 35 20
1 50 44 16 8
1 9 25 26 15
1 39 106 36 120
1 41 12 50 6
