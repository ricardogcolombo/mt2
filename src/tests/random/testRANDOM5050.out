0.526535
0.705567
0.375000
0.541485
0.735958
0.485445
0.505710
0.549695
0.645425
0.331346
0.494039
0.549846
0.199413
0.515347
0.500000
0.282919
0.367619
0.599798
0.500000
0.406139
0.707873
0.443747
0.426349
0.445668
0.671782
0.298780
0.325721
0.389604
0.680495
0.602857
0.625000
0.350775
0.349898
0.635380
0.372234
0.625000
0.779048
0.392297
0.375000
0.500000
0.662084
0.616702
0.283592
0.367619
0.735536
0.463847
0.644810
0.402624
0.522139
0.486251
